examples/spark-app-demo/k8s/liminal.yml

```diff
@@ -20,7 +20,7 @@ name: MyFirstLiminalSparkApp
 super: InfraSpark
```

Comment:
Can you update the code snippet in the doc? (docs/getting-started/spark-app-demo/)